backend/accounts/templates/email_verification.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+<body>
+    <p>Hello {{ user.username }},</p>
+    
+    <p>Please verify your email by clicking on the following link:</p>
+    <p><a href="{{ verification_url }}">{{ verification_url }}</a></p>
```

Comment:
change `<p><a href="{{ verification_url }}">{{ verification_url }}</a></p>` to `<p><a href="{{ verification_url }}">verify</a></p>`

or something better